front/migrations/20241211_parents_migrator.ts

```diff
@@ -147,7 +172,23 @@ const migrators: Record<ConnectorProvider, ProviderMigrator | null> = {
   snowflake: null,
   webcrawler: null,
   zendesk: null,
-  confluence: null,
+  confluence: {
+    transformer: (nodeId, parents) => {
+      return [
+        ...new Set([...parents, ...parents.map(convertConfluenceOldIdToNewId)]),
```

Comment:
We want the mapped parents first so that parents[0] = nodeId right?

Reply:
fixed